README.md

```diff
@@ -308,6 +308,18 @@ The task also provides following options:
   // $generatedFilesBaseDir/$sourceSet. Default is false.
   // See --descriptor_set_out in protoc documentation about what it is.
   task.generateDescriptorSet = true
+
+  // Allows to override the default for the descriptor set location
+  task.descriptorSetPath = 
+    "${projectDir}/build/descriptors/{$task.sourceSet.name}.dsc
+
+  // If true, the descriptor set will contain line number information 
+  // and comments. Default is false.
+  task.includeSourceInfoInDescriptorSet = true
```

Comment:
Can we make the options more structured, e.g.,
```
task.descriptorSetOptions.includeSourceInfo = true
task.descriptorSetOptions.includeImports = true
```

This can be done by adding a `DescriptorSetOptions getDescriptorSetOptions()` to the task class.